ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -292,142 +300,43 @@ Edit a transition to define its settings.
-  The *Import JSON* window with the JSON code for the current workflow is displayed.
-
-  ![Import JSON](../../Assets/Screenshots/ActindoWorkFlow/Workflows/ImportJSON.png "[Import JSON]")
-
-5. Select the complete JSON code in the window, replace it by the JSON code in your clipboard and click the [IMPORT JSON] button in the bottom right corner.
-  The *Import JSON* window is closed. The copied workflow is displayed in the workflow editor.
+    + Click the *Long description* field if you want to document the process step you added.   
+    The *Edit long description* window is displayed. You can add a long text using a simple markdown editor. Keep in mind that there is a limit to the amount of space that can be displayed on the screen without scrolling. 
 
-  ![Workflow editor copied](../../Assets/Screenshots/ActindoWorkFlow/Workflows/WorkflowEditorCopied.png "[Workflow editor copied]")
+        ![Edit long description](../../Assets/Screenshots/ActindoWorkFlow/Workflows/EditLongDescription.png "[Edit long description]")
 
-  > [Info] You can edit the workflow as desired before deploying it.
+        Click the [Apply changes] button to add your documentation.
 
-6. Click the ![Points](../../Assets/Icons/Points02.png "[Points]") (Points) button in the upper left corner next to the workflow name.   
-  The workflow context menu is displayed.
+    + Click the *Task event* drop-down list and select an event connected to the transition. The event is triggered and the corresponding task is assigned. For example, a process action has failed and an email is sent to all persons assigned to the event.   
+    All available task events are displayed in the drop-down list. By default, the *ActindoWorkflow process execution failed* event is preselected. The drop-down list is only displayed when the *Task* module is installed and the current user has the required rights to configure it. For detailed information, see [Manage the events](../../Tasks/Operation/01_ManageEvents.md) in the *Tasks* documentation.
 
-  ![Context menu](../../Assets/Screenshots/ActindoWorkFlow/Workflows/ContextMenu.png "[Context menu]")
+    + Edit the field(s) in the *Configuration* section to define further configuration settings to the selected transition.     
+    This section is only displayed for certain transitions. The fields displayed in the *Configuration* section depend on the selected transition. 
 
-7. Click the *Deploy* menu entry in the context menu.   
-  The copied workflow has been deployed and published.
+    + Click the ![Add](../../Assets/Icons/Plus04.png "[Add]") (Add) button right to an input name in the *Static inputs* section to add a static data value to the corresponding input. For detailed information on static inputs, see [Insert a static input](06_InsertStaticInput.md).
+    + Click the ![Delete](../../Assets/Icons/Trash07.png "[Delete]") (Delete) button right to a value to delete it. This section is only displayed for certain transitions. 
 
 
 
```

Comment:
Zu den Text in Publish / Unpublish a workflow version: "a workflow is automatically published when it is created" / "a created workflow is automatically published" -> Ich bin mir nicht sicher, ob das stimmt. Vielleicht hat es vorher bestimmt. Sofern gesehen/getestet, man muss auf Deploy (Drei-Punkte-Menü) klicken, damit es published wird. Oder dann den jeweiligen Workflow version auswählen und auf PUBLISH (editing toolbar) klicken.

Deploy vs. Publish???

Reply:
S. Info dazu z.B. hier: 

https://actindo.atlassian.net/wiki/spaces/DEVELOPMEN/pages/332366036/Process+Orchestration

![image](https://github.com/Actindo-AG/Documentation/assets/99268392/8a147f98-e531-413a-96fb-6d83071748c6)

Reply:
REiehenfolge geändert -> erst unpublish, dann publish